ros2node/ros2node/verb/info.py

```diff
@@ -46,7 +46,12 @@ def add_arguments(self, parser, cli_name):
     def main(self, *, args):
         with NodeStrategy(args) as node:
             node_names = get_node_names(node=node, include_hidden_nodes=args.include_hidden)
-        if args.node_name in (n.full_name for n in node_names):
+        count = [n.full_name for n in node_names].count(args.node_name)
+        if count > 1:
+            print('# WARNING: There are {} nodes in the graph with this exact name "{}". '
+                  'You are seeing information about only one of them.'.format(
+                    count, args.node_name))
```

Comment:
@emersonknapp consider printing to `stderr` instead of `stdout`.

Reply:
good idea